src/IIIFPresentation/API/Infrastructure/Filters/VaryHeaderAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Net.Http.Headers;
+
+namespace API.Infrastructure.Filters;
+
+public class VaryHeaderAttribute : ActionFilterAttribute
+{
+    private static readonly string[] VaryHeaders = ["X-IIIF-CS-Show-Extras", "Authorization"];
+
+    public override void OnActionExecuted(ActionExecutedContext context)
+    {
+        if (context.Result is not null)
+            context.HttpContext.Response.Headers.Append(HeaderNames.Vary, VaryHeaders);
```

Comment:
As a coding style, I think we wrap single lines like this in `{}`

Reply:
I think you can copy the style from https://github.com/dlcs/protagonist/blob/main/src/protagonist/.editorconfig - my IDE will then force me to do it :D